paddle/fluid/operators/ngraph/ngraph_engine.cc

```diff
@@ -134,12 +134,11 @@ static std::vector<std::vector<int>> NgraphOpIntervals(
   int pivot = left;
   while (pivot < right) {
     auto op_type = ops->at(pivot)->Type();
-    if (NgraphBridge::isRegister(op_type)) {
+    if (!NgraphBridge::isSupported(ops->at(pivot))) {
```

Comment:
any use about `isRegister` then?

Reply:
We created a modifed version of it, which considers and has much more conditions. On the whole, the function isRegister could be left in ngraph bridge file.

Reply:
ok